client/src/components/Inputs/BasicInput/index.js

```diff
@@ -79,6 +81,17 @@ const BasicInput = ({
           {error}
         </T.P>
       )}
+      {showForgetPassword && (
+        <T.Link
+          to={GENERAL.FORGET_PASSWORD}
+          color="neutralDark"
+          style={{ display: 'block' }}
+          mt="3"
+          ml="2"
+        >
+          Forget password?
+        </T.Link>
+      )}
```

Comment:
keep this out of the common component please, we agreed to make common components small and less logic inside

Reply:
done